### PR TITLE
[Semantic Segmentation] Add Tutorial

### PR DESCRIPTION
**Pull request by iizukak**
## Motivation and Context

This PR implement #139 .

## Description

Add tutorial for Semantic Segmentation.
Related PR is #93 . #93 implement the interface for Segmentation.

Here is the preview.
https://deploy-preview-174--blueoil.netlify.com/tutorial/image_seg.html

## How has this been tested?

I tested init, train and convert with #93 branch following this tutorial.
And I ask @lm-jira san to test this Tutorial.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature / Optimization (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/tutorial/image_seg.md [hunk 1]
```diff
@@ -1,3 +1,168 @@
...
+
+<img src="../_static/camvid.jpg" width="600">
+
+## Preparation
+
+The CamVid dataset is available from SegNet tutorial on [Github](https://github.com/alexgkendall/SegNet-Tutorial), but the dataset contain fixed absolute path of the data like [`/SegNet/CamVid/test/0001TP_008550.png`](https://github.com/alexgkendall/SegNet-Tutorial/blob/master/CamVid/test.txt). We  changed the absolute paths to relative paths. You can download from our mirror site.
+
+```
+$ wget https://s3-ap-northeast-1.amazonaws.com/leapmind-public-storage/datasets/camvid.tgz
+$ tar xzf camvid
```
Comment (lm-jira) [inline docs/tutorial/image_seg.md]: `tar xzf camvid` -> `tar xzf camvid.tgz`

Comment (lm-jira) [reply docs/tutorial/image_seg.md]: @iizukak This is the `tar xzf camvid` where I mentioned to you.

**Comment by lm-jira** [comment]
The below figure shows the accuracy of training using this tutorial.

screen shot 2019-03-04 at 12 20 50

**Comment by iizukak** [comment]
@lm-jira Thank you for testing 👍

File: docs/tutorial/image_seg.md (hunk 1 above)
Comment (iizukak) [reply docs/tutorial/image_seg.md]: @lm-jira Thanks. Fixed.

File: docs/tutorial/image_seg.md [hunk 2]
```diff
@@ -1,3 +1,168 @@
...
+This guide trains a neural network model to semantic segmentation of [CamVid](http://mi.eng.cam.ac.uk/research/projects/VideoRec/CamVid/) dataset on GPU server.
+
+<img src="../_static/camvid.jpg" width="600">
+
+## Preparation
+
+The CamVid dataset is available from SegNet tutorial on [Github](https://github.com/alexgkendall/SegNet-Tutorial), but the dataset contain fixed absolute path of the data like [`/SegNet/CamVid/test/0001TP_008550.png`](https://github.com/alexgkendall/SegNet-Tutorial/blob/master/CamVid/test.txt). We  changed the absolute paths to relative paths. You can download from our mirror site.
+
+```
+$ wget https://s3-ap-northeast-1.amazonaws.com/leapmind-public-storage/datasets/camvid.tgz
```
Comment (ruimashita) [inline docs/tutorial/image_seg.md]: I couldn't access this url `https://s3-ap-northeast-1.amazonaws.com/leapmind-public-storage/datasets/camvid.tgz`, it response 403 error.

Screen Shot 2019-03-13 at 18 57 31

File: docs/tutorial/image_seg.md [hunk 3]
```diff
@@ -1,3 +1,168 @@
...
+ │   │   ├── lib_arm.so
+ │   │   ├── lib_fpga.so
+ │   │   └── lib_x86.so
+ │   └── meta.yaml (model configuration)
+ ├── python
+ │   ├── lmnet (include pre-process/post-process)
+ │   ├── README.md
+ │   ├── requirements.txt
+ │   ├── run.py (inference script)
+ │   └── usb_camera_demo.py (demo script for object detection)
```
Comment (ruimashita) [inline docs/tutorial/image_seg.md]: `usb_camera_demo.py (demo script for object detection)` -> `usb_camera_demo.py (demo script for object detection and classification)` looks better following other tutorials.

Comment (iizukak) [reply docs/tutorial/image_seg.md]: Fixed.

File: docs/tutorial/image_seg.md (hunk 2 above)
Comment (iizukak) [reply docs/tutorial/image_seg.md]: Sorry... I'm now investigating...

Comment (iizukak) [reply docs/tutorial/image_seg.md]: @ruimashita Fixed now.

**Pull request merged**